js/components/colorSelector.js

```diff
@@ -16,12 +16,28 @@ var ColorSelector = React.createClass({
 
   render: function() {
     var colorItems = [];
+
+    var selectedColorStyle =  "solid ";
+    selectedColorStyle += this.props.skinConfig.controlBar.iconStyle.active.color ? 
+                           this.props.skinConfig.controlBar.iconStyle.active.color : 
+                           this.props.skinConfig.general.accentColor;
+
+    var activeColorStyle = { border: selectedColorStyle};
+
     for (var i = 0; i < this.props.colors.length; i++) {
-      colorItems.push(
-        <div className={this.setClassname(this.props.colors[i])} key={i}>
-          <a className={"oo-color-item oo-color-item-" + this.props.colors[i]} onClick={this.handleColorSelection.bind(this, this.props.colors[i])}></a>
-        </div>
-      );
+      if( this.setClassname(this.props.colors[i]) === 'oo-item oo-item-selected') {
```

Comment:
please simplify this conditional statement, there is no need to duplicate logic.

See comment in `fontSizeTab.js` for more details.